GovUk.Frontend.AspNetCore.Extensions/wwwroot/govuk/govuk-validation.js

```diff
@@ -30,10 +30,19 @@ function createGovUkValidator() {
     },
 
     /**
-     * Ensures the page title is prefixed with 'Error: ' when there is at least one .govuk-error-message displayed
+     * Ensures the page title is prefixed with the data-govuk-error-prefix attribute, or 'Error: ' when there is at least one .govuk-error-message displayed
      */
-    updateTitle: function () {
-      const prefix = "Error: ";
+      updateTitle: function () {
+          const titleTag = document.getElementsByTagName("title")[0];
+          const getPrefix = titleTag.getAttribute("gfa-error-prefix");
```

Comment:
You don't need to check for `gfa-error-prefix` as it's a server-side attribute that will never be there when JavaScript sees it.